src/main/java/com/powsybl/openloadflow/network/impl/PropagatedContingency.java

```diff
@@ -292,6 +292,11 @@ public Optional<LfContingency> toLfContingency(LfNetwork network) {
         Set<LfBus> buses = connectivity.getVerticesRemovedFromMainComponent();
         Set<LfBranch> branches = connectivity.getEdgesRemovedFromMainComponent();
 
+        // we should manage branches open at one side.
+        for (LfBus bus : buses) {
+            bus.getBranches().stream().filter(b -> !b.isConnectedAtBothSides()).forEach(b -> branches.add(b));
```

Comment:
Maybe we should copy branches because it could be an internal data structure of connectivity?

Reply:
Fixed